kolla-ansible/ansible/triliovault_globals.yml

```diff
@@ -1,4 +1,24 @@
 ############ Following variables are user configurable ##########
+
+### Cloud Details ###
+Cloud_Admin_UserName: ""
```

Comment:
We need to follow uniform variable naming convention.
All other variables are in 'small case'. Please follow the same for newly added variables.

Reply:
updated the all vars in small case, also tested the deployment working fine.